ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { loading } from 'ember-loading';
+
+export default class ScopesScopeProjectsTargetsIndexController extends Controller {
+  // =services
+
+  @service confirm;
+  @service ipc;
+  @service router;
+  @service session;
```

Comment:
Are we using the service `session` within this file? I can not find a reference to it.

Reply:
Yes. It is used on line 53 below to get the authenticated token.

Reply:
You are right, I miss that one, and is an important one 😅